types/recharts/recharts-tests.tsx

```diff
@@ -120,7 +120,13 @@ class Component extends React.Component<{}, ComponentState> {
                             strokeOpacity={0.2}
                             fillOpacity={0.1}
                         />
-                        <Customized component={<Text x={0} y={0} width={100} height={20}>Customized element</Text>} />
+                        <Customized
+                            component={
+                                <Text x={0} y={0} width={100} height={20} className="customized-text">
+                                    Customized element
+                                </Text>
+                            }
```

Comment:
Nit: for example to render as expected it should be wrapped in a function

Reply:
Sorry, why do you think so? I have a bunch of code where this works fine.

Reply:
Here is an example - https://jsfiddle.net/kgvzhbsy/ which results in following:

![Screenshot 2020-01-09 at 13 25 09](https://user-images.githubusercontent.com/6943514/72067636-847d7180-32e3-11ea-89ee-5da51b9cebcf.png)

Also, I've found that there is already `Customized` element defined here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/40c5fe55c59beaa23ffbf3d3fdb9cb9f5244f4ab/types/recharts/recharts-tests.tsx#L34
Maybe you could just add `className` prop there?

Reply:
My bad. Fixed. Thanks.

Reply:
Awesome!